README.md

```diff
@@ -1,27 +1,158 @@
-3. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community support and a commitment to implementation.
-4. **Visibility and Tracking**: The Author of an accepted VEPs will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback
-5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement
-Includes all the relevant information, including the design and the state.
+The process aims to focus the community's efforts on prioritized pull requests, increase the review bandwidth, and
+ensure clear visibility of feature progress and associated issues. The relatively new split into SIGs requires defining
+a common process to ensure synchronization between SIGs and uniformity of the project.
+
+**Glossary**
+
+- VEP: Virtualization Enhancement Proposal
+- EF: Enhancement Freeze
+- CF: Code Freeze
+- RC: Release Candidate
+
+## Process
+
+1. **Visibility and Tracking**: The Author of VEP will open an issue to track their progress, maturity stages (alpha,
+   beta, GA), list the associated bugs, and user feedback.
+   See the [VEP issue template](https://github.com/kubevirt/enhancements/blob/main/.github/ISSUE_TEMPLATE/vep.md), 
+   each section should be filled.
+
+2. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository.
+   See the [VEP template](https://github.com/kubevirt/enhancements/blob/main/veps/NNNN-vep-template/vep.md),
+   each section needs to be filled even if the section doesn't apply to the VEP.
+
+3. **SIG Review and Collaboration**: Although each VEP can have multiple target SIGs, it needs to have a single owning SIG.
+   In case of a cross SIG feature, the most relevant SIG should be chosen.
+   The owning SIG will ultimately decide if the VEP is accepted and whether it should be tracked for the upcoming release.  
+   If so, the SIG will assign dedicated reviewers to oversee the proposal, collaborate with others SIGs as needed, and provide 
+   feedback or veto when necessary. The author and reviewers work towards merged VEP.
+
+   Each SIG (Compute, Network, Storage) should sign-off on the VEP with an LGTM from a SIG representative (chair or approver), 
+   even if the proposal doesn't affect them. The sign-off works here as acknowledgment of the changes.
+
+4. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the
+   project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community
+   support and a commitment to implementation.
```

Comment:
```suggestion
   project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community
   support for the VEP and a commitment from the VEP owner to implement the work before Code Freeze.
```
I know that we clarify this lower down in the note, but I think it's important to make it clear here as well - which hopefully this suggestion does :)